### PR TITLE
Admin endpoint to empty out a library

### PR DESCRIPTION
**Pull request by andrewconner**
@leogrim @ryanpbrewster

### CODE REVIEW

**Comment by andrewconner** [comment]
Need this for a couple user tickets I need to close out.

File: kifi-backend/modules/shoebox/conf/shoeboxService.routes [hunk 1]
```diff
@@ -612,6 +612,7 @@ POST    /admin/bookmarks/special    @com.keepit.controllers.admin.AdminBookmarks
 POST    /admin/bookmarks/reprocessNotes @com.keepit.controllers.admin.AdminBookmarksController.reprocessNotesOfKeeps(appendTagsToNote: Boolean)
 POST    /admin/bookmarks/replaceTags @com.keepit.controllers.admin.AdminBookmarksController.replaceTagOnKeeps()
 POST    /admin/bookmarks/removeTag  @com.keepit.controllers.admin.AdminBookmarksController.removeTagFromKeeps()
+DELETE  /admin/bookmarks/emptyLibrary @com.keepit.controllers.admin.AdminBookmarksController.deleteAllKeepsFromLibrary()
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/conf/shoeboxService.routes:615]: Do `DELETE` endpoints take a request body?

Comment (andrewconner) [inline kifi-backend/modules/shoebox/conf/shoeboxService.routes:615]: Yep, it's allowed in HTTP/1.1.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/conf/shoeboxService.routes:615]: http://tools.ietf.org/html/rfc7231#section-4.3.5

> A payload within a DELETE request message has no defined semantics...

which means servers can reject it if they want to, but if it accepts it, it's all good.

**Pull request merged**